### ISSUE TITLE
JAWS announces all static text in dialog when aria-describedby is not used

### ISSUE THREAD

**Issue by kabsas** [FreedomScientific/VFO-standards-support#700]
### Summary

> When aria-describedby is not used (on the element with role="dialog"), all static text is announced prior to the focused element. This issue is present with both VPC on and off.  When aria-describedby IS used, the dialog role, label, and focused element are announced as expected.
> 
> Steps to reproduce:
> 1. Launch JAWS
> 2. Go to https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/
> 3. Navigate to the "Add Delivery Address" button.
> 4. Activate the button to launch the dialog

### Expected result

> JAWS should announce "Add Delivery Address dialog, Street edit type in text"

### Actual result

> JAWS announces "Add Delivery Address dialog Street: City: State: Zip: Special instructions: For example, gate code or other information to help the driver find you Street: Edit Type in text."

#### JAWS version and build number
2023.2302.15
 
#### Operating System and version
Windows 10 Enterprise 19044.2604

#### Browser and version:
Chrome 110.0.5481.177
Edge 110.0.1587.56 
Firefox 110.0

**Comment by JAWS-test** [comment]
Duplicate of https://github.com/FreedomScientific/VFO-standards-support/issues/662

**Comment by stevefaulkner** [comment]
closing as duplicate of #662
